README.md

```diff
@@ -48,9 +48,11 @@ You can include the Meeting Guide within any Django Template. Here is an example
 {% endblock content %}
 ```
 
+> **Note:** `wagtail-meeting-guide` does not come with the template for `meeting_guide/location.html` (the template for the `Location` page). Use the code above in your own template as needed.
+
 ## More Settings
 
-You can modify the `MEETING_GUIDE` setting in Django's settings to change the defaults in [https://github.com/code4recovery/tsml-ui#advanced-customization](tsml-ui). Here is an example settings block:
```

Comment:
In my fork this linked to the `tsml-ui` repo, which I don't have forked yet haha. Full URL was added.